model/lib/extensions/presentation_email_extension.dart

```diff
@@ -291,4 +291,11 @@ extension PresentationEmailExtension on PresentationEmail {
       type: MediaType.parse(Constant.octetStreamMimeType)
     );
   }
+
+  String? _sanitizeSearchSnippet(String? searchSnippet) => searchSnippet
+    ?.replaceAll('\r', '')
+    .replaceAll('\n', '')
+    .replaceAll('\t', '');
```

Comment:
why do you need remove this characters? Write unit test for it

Reply:
Because server send it to the client, and if we use it as is, the text will be displayed in multiple lines, and because we only allow to display subject and preview in 1 line, the text in the new lines will be missing